chart/flux/README.md

```diff
@@ -185,43 +185,51 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | ------------------------------- | ------------------------------------------ | ---------------------------------------------------------- |
 | `image.repository` | Image repository | `quay.io/weaveworks/flux`
 | `image.tag` | Image tag | `<VERSION>`
+| `replicaCount` | Number of flux pods to deploy | `1`
```

Comment:
Running more than one is not desirable -- is this so you can dial it down to none?

Reply:
Oh, this is just the documentation :-S Hmm

Reply:
If it's not meant to be an option, it should be removed as such. In other words, it can be fixed in the template as 1. Otherwise it is an option and should be documented.

Reply:
added note `more than one is not desirable`